.github/workflows/build-package-docs.yaml

```diff
@@ -31,6 +31,11 @@ on:
         - '11'
         - '10'
         - '9'
+      generate-redirects:
+        description: Generate page redirects
+        type: boolean
+        required: true
+        default: true
```

Comment:
It doesn't make sense to have a default when it's required. Use one or the other.